lib/sycamore/sycamore/writer.py

```diff
@@ -384,6 +384,10 @@ def duckdb(self, db_url=None, table_name=None, csv_directory_location=None, **kw
         ddb = DuckDBDocumentWriter(
             self.plan, client_params=client_params, target_params=target_params, name="duck_write_documents", **kwargs
         )
+        if not execute:
+            from sycamore.docset import DocSet
+
+            return DocSet(self.context, ddb)
```

Comment:
this would need a phase where it imports into the table right?

Reply:
Yep, included one as discussed